### PR TITLE
Model relationships

### PR DESCRIPTION
**Pull request by hannesqvarnstrom**
this does NOT close #14........ this opens #14

### CODE REVIEW

File: database/factories/TitleFactory.php [hunk 1]
```diff
@@ -22,7 +22,9 @@ class TitleFactory extends Factory
     public function definition()
     {
         return [
-            'title' => $this->faker->streetName
+            'title' => $this->faker->streetName,
+            'genre_id' => \App\Models\Genre::inRandomOrder()->value('id'), //PRIMARY GENRE
+            'user_id' => \App\Models\User::inRandomOrder()->value('id'), //is this needed?
```
Comment (Timpan4) [inline database/factories/TitleFactory.php:27]: What usage could it have?

**Review by Timpan4 — CHANGES_REQUESTED** [review]

File: database/factories/TitleFactory.php (hunk 1 above)
Comment (hannesqvarnstrom) [reply database/factories/TitleFactory.php:27]: I believe the 'title' column is the name of the movie or show! :) so it needs to remain i think?

Comment (Timpan4) [reply database/factories/TitleFactory.php:27]: meant more the "user_id" the //is this needed?

Comment (hannesqvarnstrom) [reply database/factories/TitleFactory.php:27]: oh yeah, i mean maybe it's fine to keep! if we have multiple admins, could be useful to have a record about which admin made what.

**Pull request merged**